test/shimmed.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+// for test:
+
+// modify es-abstract/helpers/getInferredName = null
+
+// delete global.Symbol.prototype.description;
+
```

Comment:
i don't think we need these comments here; CI will run in appropriate node versions to exercise it.
```suggestion
```